_layouts/location.html

```diff
@@ -406,7 +432,7 @@ <h4 class="card-title font-family-condensed-extra text-uppercase">
       </div>
       <div class="card">
         <a href="https://itunes.apple.com/us/app/crossroads-anywhere/id1029478803?mt=8" class="block">
-          <img alt="Download the App" class="card-img-top card-img-constrained" src="https://crds-cms-uploads.imgix.net/Uploads/crossroads-church-sites-app.jpg?auto=format&amp;crop=top&amp;fit=clamp&amp;ixjsv=2.2.3&amp;w=27" data-optimize-img>
+          <img alt="Download the App" class="card-img-top card-img-constrained" src="https://crds-media.imgix.net/y3a9myzsdjan/6CAprwzhiH5x2lOdYwP5N6/ac996756bc212dc48baede8c37891621/search-widget-crossroads-app.jpg?auto=format&amp;crop=top&amp;fit=clamp&amp;ixjsv=2.2.3&amp;w=27" data-optimize-img>
```

Comment:
I think the `&amp;` is going to give the full-sized image here not a smaller image.

- [With `&amp;` instead of `&`](https://crds-media.imgix.net/y3a9myzsdjan/6CAprwzhiH5x2lOdYwP5N6/ac996756bc212dc48baede8c37891621/search-widget-crossroads-app.jpg?auto=format&amp;crop=top&amp;fit=clamp&amp;ixjsv=2.2.3&amp;w=27)
- [With `&` instead of `&amp;`](https://crds-media.imgix.net/y3a9myzsdjan/6CAprwzhiH5x2lOdYwP5N6/ac996756bc212dc48baede8c37891621/search-widget-crossroads-app.jpg?auto=format&crop=top&fit=clamp&ixjsv=2.2.3&w=27)